### PR TITLE
Lung Rescue Things

### PR DESCRIPTION
**Pull request by Geevies**
* Having an uninjured but rescued (jabbed with a syringe to reinflate) lung will still suffocate you slowly, and give a unique message.
* Rescued lungs can now be diagnosed by stethoscopes and advanced bodyscanners.
* You can no longer rescue lungs through armor.
* Rescuing someone's lung without giving them painkillers first will be very painful for them.
* Added instructions on how to rescue lungs to the syringe extra info.
* Rescued lungs can only be fixed by doing surgery on them, or using inhaling Pneumalin.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/organs/internal/lungs.dm [hunk 1]
```diff
@@ -65,11 +65,17 @@
...
-	if(is_bruised() && rescued)
-		if(prob(4))
-			to_chat(owner, SPAN_WARNING("It feels hard to breathe..."))
-			if (owner.losebreath < 5)
-				owner.losebreath = min(owner.losebreath + 1, 5) // it's still not good, but it's much better than an untreated collapsed lung
+	if(rescued)
+		if(is_bruised())
+			if(prob(4))
+				to_chat(owner, SPAN_WARNING("It feels hard to breathe..."))
+				if (owner.losebreath < 5)
```
Comment (NonQueueingMatt) [inline code/modules/organs/internal/lungs.dm:72]: extra whitespace?

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 2]
```diff
@@ -120,17 +121,20 @@
...
 				return
-			user.visible_message(SPAN_WARNING("[user] jabs \the [src] between [P] ribs with \the [src]!"), SPAN_WARNING("You jab \the [src] between [SM] ribs!"))
-			if(H.is_lung_ruptured())
-				var/obj/item/organ/internal/lungs/L = H.internal_organs_by_name[BP_LUNGS]
-				if(!L.rescued)
-					L.rescued = TRUE
-				else
-					L.rescued = FALSE
-					L.take_damage(3)
+			var/blocked = H.getarmor_organ(H.organs_by_name[BP_CHEST], "melee")
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/syringes.dm:127]: what's with all the extra whitespace being added randomly

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 3]
```diff
@@ -120,17 +121,20 @@
...
+			if(blocked > 20)
+				user.visible_message("<b>[user]</b> jabs \the [src] into [H], but their armor blocks it!", SPAN_WARNING("You jab \the [src] into [H], but their armor blocks it!"))
+				return
+			user.visible_message("<b>[user]</b> jabs \the [src] between [P] ribs with \the [src]!", SPAN_NOTICE("You jab \the [src] between [SM] ribs!"))
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/syringes.dm:131]: "User jabs the syringe between their ribs with the syringe"?
```suggestion
			user.visible_message("<b>[user]</b> jabs \the [src] between [P] ribs!", SPAN_NOTICE("You jab \the [src] between [SM] ribs!"))
```

**Pull request merged**